app/controllers/api/v1/reviews_controller.rb

```diff
@@ -0,0 +1,16 @@
+class Api::V1::ReviewsController < ApplicationController
+
+  def create
+    review = Review.new(review_params)
+    review.location_id = params[:location_id]
+    review.user_id = current_user.id
+    if review.save
+      render json: review
+    end
+  end
```

Comment:
we need error handling in case the review does not save

just return an error message via json to show in the console BUT even better would be to take the error message you return and show it on the React page

Reply:
you're right 😁